README.md

```diff
@@ -202,6 +202,28 @@ app.use(
     },
   })
 );
+
+// Sets the `frame-ancestors` directive to "'none'"
+// similar to `helmet.frameguard({ action: "deny" })`
+app.use(
+  helmet.contentSecurityPolicy({
+    useDefaults: true,
+    directives: {
+      frameAncestors: ["'none'"],
+    },
+  })
+);
+
+// Sets the `frame-ancestors` directive to "'self'"
+// similar to `helmet.frameguard({ action: "sameorigin" })`
+app.use(
+  helmet.contentSecurityPolicy({
+    useDefaults: true,
+    directives: {
+      frameAncestors: ["'self'"],
+    },
+  })
+);
```

Comment:
I spent about an hour and a half reading helmet doc and CSP spec to write that, so I thought if it could save someone else an hour, it is worth.

I also like to have pointers to have specific alternatives to point to.

I did also feel that the section is growing a bit too long for comfort though. Let me get rid of the `'self'` version and just explain how to get to it by replacing `'none'` in the comment.